### PR TITLE
endpoints for tabs

### PR DESCRIPTION
**Pull request by StelFoog**
added endpoints for tabs

### CODE REVIEW

File: util/db/reports.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export const REPORTS_PER_REQUEST = 10;
```
Comment (HannesSundin) [inline util/db/reports.js:1]: why separate files for these constants?

File: util/db/tabs.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export const TABS_PER_REQUEST = 10;
```
Comment (HannesSundin) [inline util/db/tabs.js:1]: why separate files for these constants?

**Review by HannesSundin — APPROVED** [review]

File: util/db/reports.js (hunk 1 above)
Comment (StelFoog) [reply util/db/reports.js:1]: Fixing this in #31

File: util/db/tabs.js (hunk 2 above)
Comment (StelFoog) [reply util/db/tabs.js:1]: Fixing this in #31

**Pull request merged**